extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -117,13 +122,18 @@ define('ext.wikia.adEngine.video.googleIma', [
 						self.adDisplayContainer.initialize();
 						self.adsManager.init(width, height, google.ima.ViewMode.NORMAL);
 						self.adsManager.start();
+						console.log('***** VIDEO PLAY STARTED');
 						log('Video play: stared', log.levels.debug, logGroup);
 					};
 
 				if (this.isAdsManagerLoaded) {
+					console.log('***** ADS MANAGER LOADED');
 					callback();
 				} else if (!browserDetect.isMobile()) { // ADEN-4275 quick fix
 					log(['Video play: waiting for full load of adsManager'], log.levels.debug, logGroup);
+					console.log('***** WAITING FOR ADS MANAGER');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)